client/flask/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.7.4-alpine
```

Comment:
I think the `client` folder is dedicated to react app

Reply:
Ok. I have changed it. I've created directory `docker`. This directory is for docker setup and your flask and react apps